source/administration/config-settings.rst

```diff
@@ -3344,6 +3344,17 @@ This setting only takes effect if you are using the built-in server binary direc
 | This feature's ``config.json`` setting is ``"TLSMinVer": "1.2"`` with string input. |
 +-------------------------------------------------------------------------------------+
 
+Trusted Proxy IP Header
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specified headers that will be checked one by one for IP addresses (order is important). All other headers are ignored.
+
+New configs after v5.12 will have this set by default to ``[]``, meaning that no header will be trusted. Configs prior to v5.12 without the config entry will have it set to ``X-Forwarded-By``, ``X-Real-Ip``.
```

Comment:
Why? Are we now automatically trusting ``X-Forwarded-By``, ``X-Real-Ip`` now?

Reply:
Added more content based on the original PR.
@DSchalla can you please confirm that I did not lose any of the original meaning in my edits?